CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+`Date`
+
+- **Added `UserIconGroup` component** - stacked/non-stacked, animated lists with custom `.iui-user-icon-count` icon behavior
```

Comment:
Don't need to mention classname here. Users only care about the component and props.